### PR TITLE
VFB-163: Update lists seed

### PR DESCRIPTION
**Pull request by AmeliaIsCoding**
## What's changed
Update seeded lists data to be consistent with the spreadsheet given by client.

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| image | image|

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

If you have made any changes to the database...
  - [x] The migration files are up-to-date with my final set up (`npx supabase db diff -f <name_of_migration>` should create nothing at this point)
  - [x] I have updated the typescript definitions for the database with `db:local:generate_types`
  - [x] I have modified the seed in `seed.mts` if appropriate
  - [x] If I have modified the seed, I have also generated the seed with `npm run db:generate_seed` 
  - [x] With my final set up, I can run `npm run dev:reset_supabase` without any errors.
  - Does this require resetting the deployed database? - YES / NO (remove as appropriate)

### CODE REVIEW

**Review by AmeliaIsCoding — COMMENTED** [review]
self-review

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
 
       - name: Remove files irrelevant files that may cause TypeScript errors
         run:
-          rm -f ./seed.mts
+          rm -f supabase/seed/seed.mts
```
Comment (rickaw) [inline .github/workflows/checks.yml:24]: We should do this in other places too (other workflows & readme). Could you ctr + shift + f `seed.mts` please?

**Review by rickaw — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ REPLACE_THIS_LINE
 If you have made any changes to the database...
   - [ ] The migration files are up-to-date with my final set up (`npx supabase db diff -f <name_of_migration>` should create nothing at this point)
   - [ ] I have updated the typescript definitions for the database with `db:local:generate_types`
-  - [ ] I have modified the seed in `seed.mts` if appropriate
+  - [ ] I have modified the seed in `supabase/seed/seed.mts` if appropriate
```
Comment (rickaw) [inline .github/pull_request_template.md:21]: Could you change the `seed.mts` mentioned in `amplify.yml` too please?

File: .github/workflows/checks.yml (hunk 1 above)
Comment (rickaw) [inline .github/workflows/checks.yml:24]: just to be consisent
```suggestion
          rm -f ./supabase/seed/seed.mts
```

**Review by rickaw — APPROVED** [review]
Thanks! Feel free to merge once the tiny comments are addressed

**Pull request merged**